ext/compliance_warden.sh

```diff
@@ -5,7 +5,8 @@
 
 set -e
 
-git clone -b v33 --depth 1 https://github.com/gpac/ComplianceWarden.git
+git clone https://github.com/gpac/ComplianceWarden.git
+cd ComplianceWarden && git checkout e26973641f747cf3282004dcfb0747881207e748 && cd ..
```

Comment:
Optional: We can stay inside the `ComplianceWarden` directory, do all the work, and then `cd ..` at the end of the script. This allows us to omit `ComplianceWarden/` from all the pathnames.

This can be done separately because it is unrelated to the version bump.